src/utils.ts

```diff
@@ -53,4 +55,12 @@ export const isCurrentVersionSmallerThan = async ({
   const { major, minor, patch } = await identity.getVersion();
   const currentVersion = `${major}.${minor}.${patch}`;
   return smallerVersion({ currentVersion, minVersion: version });
-};
\ No newline at end of file
+};
+
+export const hasValidStake = (neuron: NeuronInfo): boolean =>
+  // Ignore if we can't validate the stake
+  neuron.fullNeuron !== undefined
```

Comment:
not related, you could use `@dfinity/utils` to use `isNullish` 😉

Reply:
actually that could be a good test because not sure the lib will build seamlessly in nodejs, one of these days if you got a bit of time

Reply:
I thought so, but it's not in `@dfinity/utils`, rather in nns-dapp. We might want to move them.

Reply:
Oh, someone shared problems with the cli.

That must be it...

I'll check out the new version, test it. If it doesn't work, I'll try removing utils and publish again.